### PR TITLE
app-text/zathura-cb: bump to v0.1.10

### PR DESCRIPTION
**Pull request by N-R-K**

### CODE REVIEW

File: app-text/zathura-cb/zathura-cb-0.1.10.ebuild [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	x11-libs/gtk+:3"
+
+RDEPEND="${DEPEND}"
+
+BDEPEND="virtual/pkgconfig"
+
+pkg_postinst() {
+	xdg_desktop_database_update
+
+	optfeature "jpeg support" x11-libs/gdk-pixbuf[jpeg]
```
Comment (juippis) [inline app-text/zathura-cb/zathura-cb-0.1.10.ebuild:38]: I would say always to depend on this.

**Review by juippis — APPROVED** [review]

File: app-text/zathura-cb/zathura-cb-0.1.10.ebuild (hunk 1 above)
Comment (N-R-K) [reply app-text/zathura-cb/zathura-cb-0.1.10.ebuild:38]: Not all comic books need jpeg, though. A lot of them use pngs.

Comment (juippis) [reply app-text/zathura-cb/zathura-cb-0.1.10.ebuild:38]: Are you saying there's a graphical system with png but no jpeg support? :)

I'm gonna "fix" it in a follow-up commit.

Comment (N-R-K) [reply app-text/zathura-cb/zathura-cb-0.1.10.ebuild:38]: > Are you saying there's a graphical system with png but no jpeg support? :)

No, but if someone doesn't want to enable `jpeg` on gdk-pixbuf, I see no reason to force it :)

**Pull request closed without merging**